isogram.md

```diff
@@ -0,0 +1,6 @@
+"Determine if a word or sentence is an isogram.
+An isogram (also known as a "nonpattern word") is a word or phrase without a repeating letter.
+Examples of isograms
```

Comment:
I think this needs a `:`

Also, the site currently has a problem with displaying lists if there is not blank line. The original markdown spec is ambiguous about this behavior, but GitHub handles it, and so people have come to expect it to work.

In the READMEs themselves it won't be a problem, but we also display the READMEs on the site, which would trigger the issue.